_source/_posts/2022-10-13-oktane-for-developers.md

```diff
@@ -0,0 +1,77 @@
+﻿---
+layout: blog_post
+title: "Hello Developers, Time to Get Your Oktane On"
+author: havi-hoffman
+by: advocate
+communities: [security] 
+description: "Oktane 2022 happens in San Francisco from November 8-10 and we'd love to meet you there at the Developer Hub. We've designed presentations and workshops with developers in mind." 
+tags: [announcement, conference] 
+tweets: 
+- ""
+- ""
+- ""
+image: 
+type: awareness
+---
+
+There are plenty of great reasons for developers to join us in San Francisco next month for Oktane 22, November 8-10, 2022. We're back together live and in person—celebrating our 10th annual identity event—and we've built a developer experience at the conference with folks like you in mind. 
+
+{% include toc.md %}
+
+## Why come to Oktane 
+
+Developers make it possible for anyone to use Okta's identity cloud to connect securely to any online technology, anywhere. Whether you support Okta in the workplace, or build on our Auth0 by Okta offering for authorizaton and authentication, whether you're a newcomer or a veteran of the IAM (Identity and Access Management) space, we'd love to meet you at Oktane. Engineers and developer advocates will share roadmaps and knowledge through talks, workshops, and 1:1 conversations. Come for the learning and stay for the good times. As always, we want to hear from you about your pain points, feature requests, and ideas for the future. 
+
+## Developer workshops and presentations
+
+Our speakers are developers from Okta and Auth0, as well as engineers from around the world who work with our platform. They'll cover best practices for protecting your front-end and back-end with OAuth. You'll learn how Okta APIs and SDKs can help take care of authentication, authorization, and user management for your application. We're bringing our latest hands-on workshops and challenges (road-tested and refined at Developer Days), as well as 1:1 consultations with solutions architects to the Oktane Developer Hub. 
+
+## Program overview 
+
+Here's an overview of [the program we've curated for developers](https://www.okta.com/oktane22/agenda/?filters=developer). Note: All times are PDT, and talks are 30 minutes. Video will be posted to the (https://www.youtube.com/oktadev) [OktaDev YouTube channel] in the weeks ahead. Stay tuned.  
+
+*Wednesday, November 9:*
+
+**11:00 am - Using React to Customize Your Auth0 User Experience** 
+Brandon Weber, CTO, Basis Theory [Basis Theory](https://basistheory.com/)
+
+React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.
```

Comment:
It seems that these might look better if the description was in a block quote. Then again, they might look funny because they have a blue background. For example:

> React with Auth0 offers a powerful way to bring CI/CD, automation, and customization to your identity workflows. With a simple app in React, we used a modern tool chain to build our identity UI, and leverage the Auth0 Deploy CLI Tool to automatically push those changes live. In this session, we'll show how we use React to style our sign-up and sign-in user flows in concert with Auth0.

Reply:
I'm going for it